### DISCUSSION TITLE
[BUG] Installer failing

### DISCUSSION THREAD

**Discussion by sgregori** [utmstack/UTMStack#368] [Q&A]
**Installing:**
`
image utmstack.azurecr.io/datasources:v10 could not be accessed on a registry to record
its digest. Each node will access utmstack.azurecr.io/datasources:v10 independently,
possibly leading to different nodes running different versions of the image.

image utmstack.azurecr.io/logstash:v10 could not be accessed on a registry to record
its digest. Each node will access utmstack.azurecr.io/logstash:v10 independently,
possibly leading to different nodes running different versions of the image.

image utmstack.azurecr.io/opensearch:v10 could not be accessed on a registry to record
its digest. Each node will access utmstack.azurecr.io/opensearch:v10 independently,
possibly leading to different nodes running different versions of the image.

image utmstack.azurecr.io/datasources:v10 could not be accessed on a registry to record
its digest. Each node will access utmstack.azurecr.io/datasources:v10 independently,
possibly leading to different nodes running different versions of the image.

image utmstack.azurecr.io/postgres:v10 could not be accessed on a registry to record
its digest. Each node will access utmstack.azurecr.io/postgres:v10 independently,
possibly leading to different nodes running different versions of the image.
`

**On syslog:**
`
Jan 24 19:28:21 utmstack dockerd[375]: time="2024-01-24T19:28:21.115023507Z" level=error msg="Handler for GET /v1.44/distribution/utmstack.azurecr.io/postgres:v10/json returned error: Head \"https://utmstack.azurecr.io/v2/postgres/manifests/v10\": unauthorized: {\"errors\":[{\"code\":\"UNAUTHORIZED\",\"message\":\"authentication required, visit https://aka.ms/acr/authorization for more information.\"}]}"
Jan 24 19:35:06 utmstack dockerd[375]: time="2024-01-24T19:35:06.819988870Z" level=info msg="Attempting next endpoint for pull after error: Head \"https://utmstack.azurecr.io/v2/logstash/manifests/v10\": unauthorized: {\"errors\":[{\"code\":\"UNAUTHORIZED\",\"message\":\"authentication required, visit https://aka.ms/acr/authorization for more information.\"}]}"
`

**To Reproduce**
Run the latest installer.

**Expected behavior**
Clean install with all docker containers created.

**Environment**
 - OS: Ubuntu 22.04 inside LXC container

Thanks.

**Comment by osmontero** [comment]
Hi @sgregori, we don't have support for LXC, try installing it on a VM with full virtualization.

**Comment by osmontero** [comment]
Hi @sgregori, please let us know if you were able to run it on a virtual machine.

**Reply by sgregori** [reply]
Hi, i tried again but there are too many problems, not just the problems described in the firs message.

Now i'm trying a another product that works nice inside LXC container, but i don't discard to continue trying utm-stack again.

I installed too, a libvirt environment inside a lxc container and i have running a windows machine, but i'm not sure if this scheme is that i want for the SOC.

Maybe i can use another server for running that.

Thanks.
